scripts/zones/Promyvion-Dem/npcs/_0id.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Promyvion Dem
+-- Memory Flux 4th floor
+-----------------------------------
+local ID = require("scripts/zones/Promyvion-Dem/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
```

Comment:
unused imports

Reply:
When obtaining a key item, I don't need to put in keyitems import?